### DISCUSSION TITLE
how does a form view work? [Nov 08]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4399] [Discord Threads]
> 🙋‍♂️ **luposung**

**Comment by o1lab** [comment]
> 🙋‍♂️ **wkw**  
#### by default, you input data in a grid view. If you create a form view, you input your data in a form format. you can drag n drop the fields to design your form.  
![image0](https://cdn.discordapp.com/attachments/958575615198527539/958576946558681200/unknown.png)  
#  
> 🙋‍♂️ **luposung**  
#### thanks. users might be used to copy-and-paste as they work with excel file, wondering how nocoDB handles this use case. in this screenshot Actor and Film is many-to-many relationship, can i firstly copy rows i want in Film then paste into the form field? or instead i select rows i want in Film and bring selected rows over the form to create an Actor?  
#  
> 🙋‍♂️ **wkw**  
#### if your initial source is on excel, then i suggest you to use the third option `create project from excel` when creating your project  
![image0](https://cdn.discordapp.com/attachments/958575615198527539/958609232100462652/unknown.png)  
#  
> 🙋‍♂️ **luposung**  
#### thanks. can we have a form with subtable like this?  
![image0](https://cdn.discordapp.com/attachments/958575615198527539/958658863467487292/IC-Purchase-Order-Template_Google.png)  
#  
> 🙋‍♂️ **wkw**  
#### not supported  
#  
> 🙋‍♂️ **luposung**  
#### got it. thanks.  
#  
> 💁 **o1lab**  
#### this would be like page designer and there is a feature request for it in github  
#  
> 🙋‍♂️ **luposung**  
#### yeh i saw it in the roadmap. looking forward to it.  
#  
> 🙋‍♂️ **FirstFloris**  
#### Is there a way to customize UI?  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/958575615198527539)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
